pkg/pipeline/write/write_loki.go

```diff
@@ -58,6 +59,11 @@ type Loki struct {
 	exitChan   chan bool
 }
 
+var recordsWritten = operationalMetrics.NewCounter(prometheus.CounterOpts{
+	Name: "write_loki_records_written",
```

Comment:
nit: on naming, simply "loki_records_written" would be more suitable?

Reply:
fixed